projects/js-packages/charts/src/components/line-chart/stories/index.stories.tsx

```diff
@@ -55,3 +52,19 @@ CustomDimensions.args = {
 	margin: { top: 20, right: 20, bottom: 30, left: 40 },
 	data: sampleData,
 };
+
+// Story with horizontal legend
+export const WithLegend = Template.bind( {} );
+WithLegend.args = {
+	...Default.args,
+	showLegend: true,
```

Comment:
Let's put this property to all default objects so we can also toggle it from the Storybook settings section.